cp/main.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <fcntl.h>
+#include <unistd.h>
+#include <stdlib.h>
+
+void quick(int er) {
+    exit(er);
+}
+
+int main(int argc, char** argv) {
+    if (argc != 3) {
+        quick(errno);
+    }
+    int file = open(argv[1] , O_RDONLY);
+    if (file < 0) {
+        std::cerr << file;
```

Comment:
Program will output "-1". What will you learn from that ? Make your logs informative.